brian2cuda/device.py

```diff
@@ -1206,6 +1087,13 @@ def build(self, directory='output',
 
         self.generate_makefile(writer, cpp_compiler, cpp_compiler_flags, nb_threads=0, disable_asserts=disable_asserts)
 
+        logger.info("Using the following preferences for CUDA standalone:")
+        for pref_name in prefs:
+            if "devices.cuda_standalone" in pref_name:
+                logger.info("{} = {}".format(pref_name,prefs[pref_name]))
+            else:
+                logger.debug("{} = {}".format(pref_name,prefs[pref_name]))
+
```

Comment:
This will generate a `debug` log which is titled as `CUDA standalone` but has all preferences available. Please split them up and title the non-CUDA preferences accordingly.

And I'm a bit picky with design here :P Could you make the log message for each preference line indented by a tab (using `\t` at the start of the those log strings).